tpp-app/src/services/OnboardingService.js

```diff
@@ -0,0 +1,107 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { FLOW_LEVEL, TRACK_SYMPTOMS, KEYS } from './utils/constants.js'
+import { initializeEmptyYear } from "./utils/helpers.js"
+import { Symptoms } from './utils/models.js';
+
+/**
+ * Initializes user's initial period length and average period length. 
+ * @param periodLength a number representing the period length
+ * @returns a promise resolving when the multiset operation is complete 
+ */
+export const PostInitialPeriodLength = async (periodLength) => new Promise( async (resolve, reject) => {
+    try {
+        if(periodLength != null) {
```

Comment:
```suggestion
        if(periodLength) {
```
Shortens it and expands the range of checks. This will make sure that it's defined in any way that's not `undefined` or `null`. Fun fact, you can use this for strings too and it'll return false if it's an empty string.